### PR TITLE
Create husky.md

### PR DESCRIPTION
**Pull request by typicode**

### CODE REVIEW

File: _projects/husky.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: husky
+description: Git hooks made easy
+repository: https://github.com/typicode/husky
+language: JavaScript
+developer:
+  name: Typicode
+  homepage: https://github.com/typicode
+version: 5.0.0
```
Comment (kemitchell) [inline _projects/husky.md]: Are you using Parity 5.0.0? The latest is [7.0.0](https://paritylicense.com/versions/7.0.0).

Comment (typicode) [reply _projects/husky.md]: My bad, I had husky v5.0 in mind when filling this field. Fixed :+1: :slightly_smiling_face:

Comment (kemitchell) [reply _projects/husky.md]: It's not you. It's confusing. Another PR did the same thing.

I will merge this, deploy, and then go back and fix the field name.

**Comment by kemitchell** [comment]
Deployed!

**Comment by typicode** [comment]
🎉

**Pull request merged**